extras/dynflow-sidekiq.rb

```diff
@@ -8,9 +8,6 @@
 rails_env_file = File.expand_path('./config/environment.rb', rails_root)
 require rails_env_file
 
-redis_url = ENV['DYNFLOW_REDIS_URL'] || SETTINGS.dig(:dynflow, :redis_url)
```

Comment:
I am not sure I understand (dig xD) why you have done this? how we should configure redis url now? :(

Reply:
Only using env variables (the last commit also changed the systemd unit). Alternatively we would need to set up reliable fetch in dynflow after the url was properly configured.